publishedbranches/docs-node.yaml

```
@@ -1,16 +1,19 @@
 prefix: 'drivers/node'
 version:
   published:
-    - 'master'
+    - '4.0 (upcoming)'
+    - '3.6 (stable)'
   active:
-    - 'master'
-  stable: 'master'
-  upcoming: master
+    - '4.0 (upcoming)'
+    - '3.6 (stable)'
+  stable: 'v3.6'
+  upcoming: 'v4.0'
 git:
   branches:
-    manual: 'master'
+    manual: 'v4.0'
```

This field should correspond to the branch name, which I presume to be `master` from line 15!
```suggestion
    manual: 'master'
```